eea/kitkat/profiles/default/metadata.xml

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <metadata>
-  <version>4.0</version>
+  <version>4.2</version>
   <dependencies>
     <dependency>profile-collective.taxonomy:default</dependency>
+    <dependency>profile-collective.volto.subsites:default</dependency>
```

Comment:
To be removed.

Reply:
done [c7b653a](https://github.com/eea/eea.kitkat/pull/19/commits/c7b653a42b6a0237af1e27f0a48578ea19e39afa)